x-pack/test/functional_with_es_ssl/apps/uptime/alert_flyout.ts

```diff
@@ -124,7 +126,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           expect(timerangeUnit).to.be('h');
           expect(timerangeCount).to.be(1);
           expect(JSON.stringify(filters)).to.eql(
-            `{"url.port":["5678"],"observer.geo.name":["mpls"],"monitor.type":["http"],"tags":[]}`
+            `{"tags":[],"url.port":["5678"],"observer.geo.name":["mpls"],"monitor.type":["http"]}`
```

Comment:
Is there a need for this change or just the result of doing some testing?

Reply:
yes, seems like the order has changed.